zach_brady/mid-term/lib/shelter.rb

```diff
@@ -0,0 +1,38 @@
+require 'pry'
+
+class Shelter
+    attr_accessor :name
+
+    def initialize(name, animals=[], clients=[])
+        @name = name
+        @animals = animals
+        @clients = clients
+    end
+
+    def get_name
+        return @name
+    end
+
+    def get_animals
+        return @animals
+    end
+
+    def get_clients
+        return @clients
+    end
+
+
+    def adopt(animal, client)
+        client.pets << animal
+    end
+
+    def return_pet(animal, client)
+        if client.pets.include? animal
+            client.pets.delete(animal)
```

Comment:
Does the animal get into the shelter or just get thrown into the Ruby garbage collector?